src/shared/components/DashboardRoute.tsx

```diff
@@ -59,30 +58,35 @@ class DashboardRoute extends PureComponent<Props> {
     const dashboardID = this.props.match.params.dashboardID
     dashboardVisit(dashboardID, new Date().getTime())
     event('Dashboard Visit', {dashboardID})
-    const urlVars = qs.parse(this.props.location.search, {
-      ignoreQueryPrefix: true,
-    })
 
     // always keep the dashboard in sync
     if (dashboard !== dashboardID) {
       updateDashboard(dashboardID)
     }
 
+    const params = new URLSearchParams(window.location.search)
+    const vars = params
+      .entries()
+      .filter(([k, _v]) => /(^vars\[)(.+)(\]$)/g.test(k))
+      .reduce((acc, [k, v]) => {
+        acc[k.slice(5, -1)] = v
```

Comment:
is the 5 here the length of `vars` whatever?

Reply:
yes